### PR TITLE
Update styles folder to introduce files for global variable names

### PR DESCRIPTION
**Pull request by mko5118**
### Description

Update styles folder to introduce global style variable names

### Changes

[describe the changes here in plain English and in tech, as if explaining to someone without context]
- Updated styles folders to implement styles.scss (base SASS file)
- Updated styles folder to implement _global.styles.scss (global SASS variables file)
- Updated index.js to import styles.scss into project
- Updated index.html to import google fonts link

### Verification

global_styles_variables

### Trello Link

https://trello.com/c/5iCweVLT/21-implement-styles-into-client-side

### Notes

Colors and font's implmented are subject to change

### CODE REVIEW

**Comment by mko5118** [comment]
Two typos noticed:
-  $primary-color-light: lighten
- $tertiary-color-light: lighten

**Comment by Formless** [comment]
Please squash your commits to reduce noise in the commit history: https://blog.carbonfive.com/2017/08/28/always-squash-and-rebase-your-git-commits/
